src/plus/gk/checkin.ts

```diff
@@ -176,7 +180,18 @@ export function getSubscriptionFromCheckIn(
 		);
 	}
 
-	if (effective == null || getSubscriptionPlanPriority(actual.id) >= getSubscriptionPlanPriority(effective.id)) {
+	const isActualLicenseExpired =
```

Comment:
We could just check `latestStatus` and/or `expiresOn` date of the `actual` here, right? Then no need to go through `computeSubscriptionState`.